editor/editor_properties_array_dict.cpp

```diff
@@ -934,6 +943,10 @@ void EditorPropertyDictionary::_change_type_menu(int p_index) {
 			update_property();
 			break;
 
+		case EditorPropertyDictionaryObject::NOT_CHANGING_TYPE:
+			ERR_FAIL_MSG("Tried to change the type of a dict key or value, but nothing was selected.");
```

Comment:
Sorry, I didn't notice this one uses `break` 🙃
It should be `ERR_PRINT` like before.

Reply:
Aaaah ok I just understood what this things are doing ><' I'm really not a C++ programmer, not used to this macros ^^'

Reply:
Should I use the variant that also notifies the editor ?

Reply:
From what I understood in another PR, `ERR_PRINT` only prints to the standard output, while `ERR_PRINT_ED` is also displaying a nice toast notification inside the editor (at bottom right corner by default if I well remember)

Reply:
This should never happen, I don't think it's really useful to display a notification for this.